src/templates/components/button.html

```diff
@@ -0,0 +1,15 @@
+<!--[if mso]>
+        <v:roundrect xmlns:v="urn:schemas-microsoft-com:vml" xmlns:w="urn:schemas-microsoft-com:office:word" href="{{ url }}" style="margin-top:24px;margin-bottom:8px;height:56px;v-text-anchor:middle;width:300px;padding:0 64px;" arcsize="50%" stroke="f" fillcolor="#161422">
+            <w:anchorlock/>
+            <center style="color:#ffffff;font-family:sans-serif;font-size:16px;font-weight:bold;">
+                {{ text }}
+        </center>
+    </v:roundrect>
+<![endif]-->
+<!--[if !mso]> <!-->
```

Comment:
What's this used for?

Reply:
This file is checking whether we're in a Microsoft Office environment, and if so using their components instead... Does that help?

Reply:
Yes, thank you